superset-frontend/src/assets/images/filter-results.svg

```diff
@@ -0,0 +1,34 @@
+<!--
```

Comment:
I'm guessing that there are a handful of these empty state SVG images by now. Perhaps they should get their own directory just for content organization?